### PR TITLE
Ajit assignment 8_Done

### PR DESCRIPTION
**Pull request by AjitSJune20**

### CODE REVIEW

File: src/ajitSJune20/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package ajitSJune20;
+
+public class BankingSystem {
+
+	int debitCount = 0, creditCount = 0, printBalanceCount = 0, currentBalance = 0;
+	static int totalDebitCount = 0, totalCreditCount = 0, TotalPrintBalanceCount = 0;
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/BankingSystem.java]: Remove the initialization of all instance variables.

File: src/ajitSJune20/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	int creditAmount(int creditAmt) {
+		currentBalance = currentBalance + creditAmt;
+		System.out.println("Credit Amount : " + creditAmt);
+		totalCreditCount++;
+		return creditCount++;
+
+	}
+
+	int debitAmount(int debitAmt) {
+		if (debitAmt >= 0 && debitAmt <= currentBalance) {
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/BankingSystem.java]: Remove equal here.

File: src/maheshKJun20/Employee.java [hunk 3]
```diff
@@ -10,6 +9,7 @@
 g) Create a method called isEligibleForIncrement which will return boolean value, if deptId is 1 or salary is less then 30,000 return true and main method will print Employee is eligible for increment, otherwise print not "eligible for increment" message.
 
 */
+package maheshKJun20;
```
Comment (KrishnaKTechnocredits) [inline src/maheshKJun20/Employee.java:12]: Good job.

File: src/ajitSJune20/BankingSystem.java (hunk 2 above)
Comment (AjitSJune20) [reply src/ajitSJune20/BankingSystem.java]: Incorporated changes.

File: src/ajitSJune20/BankingSystem.java (hunk 1 above)
Comment (AjitSJune20) [reply src/ajitSJune20/BankingSystem.java]: Incorporated changes

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**